pandas/tests/reshape/concat/test_dataframe.py

```diff
@@ -167,3 +167,9 @@ def test_concat_dataframe_keys_bug(self, sort):
         # it works
         result = concat([t1, t2], axis=1, keys=["t1", "t2"], sort=sort)
         assert list(result.columns) == [("t1", "value"), ("t2", "value")]
+
+    def test_concat_dataframe_one_element_reversed(self):
```

Comment:
can you share this using the frame_or_series fixture

Reply:
Nice! Thank you, done